pkg/server/spec/container.go

```diff
@@ -5,6 +5,33 @@ import (
 	"github.com/luizalabs/teresa/pkg/server/storage"
 )
 
+type ContainerLimits struct {
+	CPU    string
+	Memory string
+}
+
+type VolumeMounts struct {
+	Name      string
+	MountPath string
+	ReadOnly  bool
+}
+
+type Port struct {
+	Name          string
+	ContainerPort int32
```

Comment:
Any specific reason for using named ports or it's just a choice?

Reply:
I guess that if we have more than 1 one port, this need to be named.